.github/workflows/koupleless_adapter_release.yml

```
@@ -0,0 +1,79 @@
+        run: |
+          echo "Finding Maven Modules"
+          # 定位 pom.xml 文件，通过 grep 排除掉根目录下的 pom.xml
+          modules=$(find . -name pom.xml -maxdepth 2 -not -path "./pom.xml" -exec dirname {} \;)
+          echo "$modules"
+          # 使用 jq 将 shell 数组转换成 JSON 数组供后续步骤使用
+          modules_in_json=$(echo $modules | jq -R -s -c 'split("\n") | map(select(length > 0))')
+          echo "$modules_in_json"
+          echo "modules-list=$modules_in_json" >> $GITHUB_OUTPUT
+
+      - name: release adapter for jdk8
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:6:44: Double quote to prevent globbing and word splitting

(shellcheck)